osquery/remote/enroll/tls_enroll.cpp

```diff
@@ -38,7 +38,7 @@ CLI_FLAG(uint64,
          "same as [config_tls_max_attempts]");
 
 /// Enrollment TLS endpoint (path) using TLS hostname.
-CLI_FLAG(string,
+FLAG(string,
```

Comment:
I'm leery of changing this

Reply:
Yes, please do not change this, it is required for configuration, chicken and egg problem so to speak.